policies/rules.md

```diff
@@ -0,0 +1,145 @@
+
+[Submit your project for the final prizes](submissions.html#final-prize-submission)
+
+### Prizes
+
+Prizes of monetary value will be distributed to winning teams electronically. All other prizes will be distributed in the manner deemed most appropriate by organizers and sponsors.
+
+Some sponsors may be given the opportunity award prizes during the course of the competition. These prizes will be communicated and awarded by sponsors in the manner they see fit, and subject to the rules and veto of the event organizers.
+
+### Judging
+
+During the final prize round a team of expert judges will review each entry. Teams entering their projects for prizes will be expected to be available to answer judges questions via email, and if possible in person.
+
+Teams may also be asked to confidentially share source code and project details with judges so that they might better evaluate a projects merits.
+
+Failure to communicate or supply judges with access to a project could impact a projects success in the competition.
+
+### Intellectual Property
+
+All teams will continue to own the intellectual rights to their work and projects during and after this competition.
+
+Projects are encouraged to open source their code where applicable, but are not required to do so. During judging teams may be asked to provide judges access to source code to verify technical and other claims. Failure to do so could disadvantage or disqualify teams during judging, at the discretion of the judging committee and organizers.
+
+Teams are welcome to use someone else's assets (e.g. libraries, frameworks, templates, source code, images, etc.) but are required to comply with the licenses for these assets. Teams may be asked to supply proof they have done so. Teams violating license and intellectual property rights of others my be disqualified and forfeit prizes awarded them.
+
+### Media Consent
+
+From time-to-time projects and other information about the Hackathon will be shared publicly via social and other media. We respect participants right to privacy and will not share personal information without prior consent.
+
+### Communication Consent and Responsibilities
+
+By participating participants agree to give organizers permission to contact them with information and communication relevant to CODEVID-19 via Slack, email, or other methods deemed appropriate by event organizers. This consent may be withdrawn at anytime be contacting contact@codevid19.com, or using the unsubscribe feature in the mailing list.
+
+Participants understand that by withdrawing their consent to communicate with organizers they will be putting themselves, their team, and their project at a competitive disadvantage. Organizers cannot be held responsible for participants receiving information supplied via email or Slack. It's participants resonsponsibiltiy to regularly check the website or Slack for updates should they choose to withdraw consent.
```

Comment:
resonsponsibiltiy -> responsibility

Reply:
It's a participant's

(add apostrophe to "participant")